aws-api/src/main/java/com/amplifyframework/api/aws/GsonErrorDeserializer.java

```diff
@@ -49,7 +50,7 @@
         String message = null;
         List<GraphQLLocation> locations = null;
         List<GraphQLPathSegment> path = null;
-        Map<String, Object> extensions = null;
+        HashMap<String, Object> extensions = null;
```

Comment:
Ordinarily the declaration is the bast `Map<KeyType, ValType>`, unless you are using some exotic property of a map that requires specification of which map type you need. For example, if you need ordering properties, you'd both instantiate and declared `LinkedHashMap`, since the type _is_ important, you're no longer dealing in "any old map." But here, I think you're just dealing with any old map.

Most common pattern is like:
```
static Map<Foo, Bar> createOne() {
    Map<Foo, Bar> fooBar = new HashMap<>(); // This is the only place the special type is mentioned
    fooBar.put("ham", "sandwich");
    return fooBar;
}
```

Reply:
Agreed!  Not sure why I specified `HashMap` instead of `Map`.  I will change to just `Map`.